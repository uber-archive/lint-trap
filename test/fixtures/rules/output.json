[
    {
        "type": "file",
        "file": "func-names-invalid.js",
        "errors": [
            {
                "type": "error",
                "linter": "eslint",
                "file": "func-names-invalid.js",
                "line": 4,
                "column": 10,
                "rule": "func-names",
                "message": "Missing function expression name."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "func-names-invalid.js",
                "line": 7,
                "column": 10,
                "rule": "space-after-function-name",
                "message": "Function name \"baz\" must not be followed by whitespace."
            }
        ]
    },
    {
        "type": "file",
        "file": "func-names-valid.js",
        "errors": []
    },
    {
        "type": "file",
        "file": "globals-invalid.js",
        "errors": [
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 9,
                "column": 8,
                "rule": "no-undef",
                "message": "'process' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 10,
                "column": 8,
                "rule": "no-undef",
                "message": "'global' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 11,
                "column": 8,
                "rule": "no-undef",
                "message": "'exports' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 12,
                "column": 8,
                "rule": "no-undef",
                "message": "'console' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 13,
                "column": 8,
                "rule": "no-undef",
                "message": "'clearInterval' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 14,
                "column": 8,
                "rule": "no-undef",
                "message": "'setInterval' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 15,
                "column": 8,
                "rule": "no-undef",
                "message": "'clearTimeout' is not defined."
            }, {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 16,
                "column": 8,
                "rule": "no-undef",
                "message": "'setTimeout' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 17,
                "column": 8,
                "rule": "no-undef",
                "message": "'Buffer' is not defined."
            }
        ]
    },
    {
        "type": "file",
        "file": "globals-valid.js",
        "errors": []
    }
]
