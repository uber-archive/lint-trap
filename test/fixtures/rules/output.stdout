func-names-invalid.js
    error  eslint     4:10   Missing function expression name.                        func-names
    error  eslint     7:10   Function name "baz" must not be followed by whitespace.  space-after-function-name

globals-invalid.js
    error  eslint     9:8    'process' is not defined.        no-undef
    error  eslint    10:8    'global' is not defined.         no-undef
    error  eslint    11:8    'exports' is not defined.        no-undef
    error  eslint    12:8    'console' is not defined.        no-undef
    error  eslint    13:8    'clearInterval' is not defined.  no-undef
    error  eslint    14:8    'setInterval' is not defined.    no-undef
    error  eslint    15:8    'clearTimeout' is not defined.   no-undef
    error  eslint    16:8    'setTimeout' is not defined.     no-undef
    error  eslint    17:8    'Buffer' is not defined.         no-undef
